tests-extension/.openshift-tests-extension/openshift_payload_olmv0.json

```diff
@@ -282,6 +282,62 @@
       "exclude": "topology==\"External\""
     }
   },
+  {
+    "name": "[sig-operator][Jira:OLM] OLMv0 should PolarionID:24916-Author:bandrade-ConnectedOnly-Medium-Operators in AllNamespaces should be granted namespace list",
+    "originalName": "[sig-operator][Jira:OLM] OLM for an end user handle common object Author:bandrade-ConnectedOnly-Medium-24916-Operators in AllNamespaces should be granted namespace list",
```

Comment:
@bandrade originalName is not case title in test-private.

if it is first migration, no need to set it. 

it is only needed when you change case title after you migrated it.